File: 1-Iniciante/ex1078.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int n, i;

    /*entrada*/
    scanf("%d", &n);

    /*processamento & saída*/
    for(i = 1; i < 11; i++){
        printf("%d x %d = %d\n", i, n, n*i);
    }

    return 0;
}

File: 1-Iniciante/ex1078.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let n = parseInt(input);

// processamento & saida
for(let i = 1; i < 11; i++) console.log(`${i} x ${n} = ${i * n}`);